library/classes.lisp

```diff
@@ -73,8 +73,8 @@
 
   (define-struct (Tuple :a :b)
     "A heterogeneous collection of items."
-    (first :a)
-    (second :b))
+    (first "The first tuple slot." :a)
```

Comment:
Find a better struct in the stdlib to add docstring to

Reply:
The only structs defined in the stdlib are:

`Tuple`
`Tuple3
`Tuple4`
`Tuple5`

Reply:
Switched `dual` to a struct, added docstrings